tests/integration/modules/test_vsphere.py

```diff
@@ -52,3 +52,13 @@ def test_list_ssds(service_instance):
     for host_name in ret:
         for ssd in ret[host_name]:
             assert isinstance(ssd, str)
+
+
+def test_list_non_ssds(service_instance):
+    ret = vsphere.list_non_ssds(
+        service_instance=service_instance,
+    )
+    assert ret
+    for host_name in ret:
+        for ssd in ret[host_name]:
+            assert isinstance(ssd, str)
```

Comment:
I was thinking about this, is there a way to check the named ssd is a non-ssd and not a real ssd ?